### ISSUE TITLE
秒记

### ISSUE THREAD

**Issue by zhaipro** [zhaipro/keras-wdsr#1]
https://drive.google.com/drive/folders/1SRq4CLb-iMve7n32Auln03jubCSA9fNq?usp=sharing

**Comment by zhaipro** [comment]
https://github.com/JiahuiYu/wdsr_ntire2018
https://github.com/krasserm/super-resolution
https://github.com/wmylxmj/Anime-Super-Resolution
https://zh.wikipedia.org/wiki/峰值信噪比
https://zh.wikipedia.org/wiki/对数
[PixelShuffler](https://gist.github.com/t-ae/6e1016cc188104d123676ccef3264981)
[K.shape vs K.int_shape](https://keras.io/zh/backend/#shape)
https://keras.io/zh/
[WDSR(NTIRE2018超分辨率冠军)【深度解析】](https://blog.csdn.net/leviopku/article/details/85048846)
https://keras.io/zh/layers/writing-your-own-keras-layers/
https://www.kancloud.cn/aollo/aolloopencv/269599

**Comment by zhaipro** [comment]
还看不懂的有…
http://lipixun.me/2018/07/08/weight-normalization
https://github.com/ychfan/tf_estimator_barebone/blob/master/common/layers.py
我总觉得这里面的写错了，啊，搞不懂。

https://github.com/krasserm/weightnorm/blob/master/keras_2/weightnorm.py
为什么要在损失函数里实现这个小功能呢？
方便为所有卷积层添加权重归一化？
可是损失函数在不断的进化，我们这样岂不是如同下车吗？

https://github.com/tensorflow/addons/blob/master/tensorflow_addons/layers/wrappers.py#L24
试试参考官方教程？

https://arxiv.org/pdf/1602.07868.pdf
https://github.com/keras-team/keras/blob/master/keras/optimizers.py#L436
https://github.com/seanpmorgan/tf-weightnorm/blob/master/normalization.py

**Comment by zhaipro** [comment]
已知问题
是我不会用cv2的resize吗？
需要增强下采样

**Comment by zhaipro** [comment]
提到日程，高清化摄像头，从而美化风格变换。
摄像头：640 480
大屏：1920 1080
